src/scripts/worldmap/MapHelper.ts

```diff
@@ -102,6 +103,8 @@ class MapHelper {
             cls = areaStatus[areaStatus.uncaughtShinyPokemon];
         } else if (!RouteHelper.isAchievementsComplete(route, region)) {
             cls = areaStatus[areaStatus.missingAchievement];
+        } else if (RouteHelper.minPokerus(RouteHelper.getAvailablePokemonList(route, region, true)) < 3 && Settings.getSetting('showPokerusMapStatus').observableValue()) {
```

Comment:
For optimization, it should check the setting before getting the list. Else it will spend CPU on calculating it for all routes, for players who have turned it off

Reply:
Removed boolean setting